README.md

```diff
@@ -96,32 +109,17 @@ If you keep having trouble, you can also run it from the command line, like this
 
 This will output a clearer log message about what the problem is.
 
-## Getting help
-If you run into trouble, or have any questions or suggestions, you can get in
-touch with Nammu's developers by creating a new issue in this repository and
-telling us about your problem [here](https://github.com/oracc/nammu/issues/new).
-
-Please follow the guidelines to help us better understand and reproduce your
-problem.
-
-## Known problems and features under development
-We are currently working on the next release of Nammu, which includes some new
-features as well as bug fixes.
-
-Here's a list of known bugs:
+### Known problems:
 * Nammu doesn't validate or lemmatise properly when filenames include a space.
```

Comment:
Yep. That's the ORACC server not liking files with spaces on their filenames. We'll be working on the backend later on in the project possible and could address this.